boxes/boxes/vite/package.json

```diff
@@ -14,14 +14,14 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@aztec/accounts": "portal:../../../yarn-project/accounts",
-    "@aztec/aztec.js": "portal:../../../yarn-project/aztec.js",
-    "@aztec/bb-prover": "link:../../../yarn-project/bb-prover",
-    "@aztec/circuit-types": "portal:../../../yarn-project/circuit-types",
-    "@aztec/key-store": "link:../../../yarn-project/key-store",
-    "@aztec/kv-store": "portal:../../../yarn-project/kv-store",
-    "@aztec/pxe": "link:../../../yarn-project/pxe",
-    "@aztec/simulator": "link:../../../yarn-project/simulator",
+    "@aztec/accounts": "latest",
+    "@aztec/aztec.js": "latest",
```

Comment:
I'm not sure on this - won't this fail to reflect changes in the repo?

Reply:
It uses the same strategy as the other boxes, via linking in the main package.json. You actually showed me that, but maybe I misunderstood!